packages/docs/src/routes/docs/overview/index.mdx

```diff
@@ -51,13 +51,17 @@ Qwik is a new kind of web framework that can deliver instant loading web applica
   </div>
   <div class="card">
     <h3>Instant-on</h3>
-    <p>Unlike other frameworks, Qwik is resumable which means Qwik applications require 0 hydration.
-      This allows Qwik apps to have instant-on interactivity, regardless of size or complexity</p>
+    <p>
+      Unlike other frameworks, Qwik is resumable which means Qwik applications require 0 hydration.
```

Comment:
believe it or not this is not valid HTML. 🤯 

```html
<p>
   Foo
```

Turns into 
```html
<p>
  <p>
   Foo
```

Nesting `<p>` tags is dissolved. When you navigate to this page the SSR will throw an error.

Reply:
@mhevery I think prettier formatted this. Will update

Reply:
@mhevery updated